_pages/business-units/operations/outreach/blogging.md

```diff
@@ -7,19 +7,38 @@ questions:
   - outreach
 ---
 
-This guide helps 18F employees through the blogging process and assists them in properly framing their post so that it’s concise, readable, and achieves its goals. If you have any questions, ask them in \#blog or by email to [18f-outreach@gsa.gov](mailto:18f-outreach@gsa.gov).
+This guide helps TTS employees through the blogging process and assists them in properly framing their post so that it’s concise, readable, and achieves its goals. If you have any questions, ask them in [#blog](https://18f.slack.com/messages/blog/) or by email to [18f-outreach@gsa.gov](mailto:18f-outreach@gsa.gov).
 
-Everyone at 18F is encouraged to contribute to [our blog](https://18f.gsa.gov/blog/). We value a diversity of voices and positions. You don’t have to be a project lead, be in management, or even be at 18F for a set amount of time to draft a post. If you’ve got an idea that you think will help advance 18F’s mission, we’d love to hear it.
+Everyone at TTS is encouraged to contribute to our blogs. We value a diversity of voices and positions. You don’t have to be a project lead, be in management, or even be at TTS for a set amount of time to draft a post. If you’ve got an idea that you think will help advance TTS’s mission, we’d love to hear it.
+
+## Choosing a blog
+
+TTS has a number of blogs that employees can publish to. Normally folks will
+publish to their team's blog. Depending on the
+content, however, it is possible that you could publish to the blog of a team that isn't
+your own. These are the TTS blogs:
+
+- [18F blog](https://18f.gsa.gov/blog/)
+- [digital.gov blog](https://digital.gov/posts/)
+- [FedRAMP blog](https://www.fedramp.gov/blog/)
+- [data.gov blog](https://www.data.gov/meta/)
+- [USAGov blog](https://blog.usa.gov/)
+- [Citizen Science blog](https://www.citizenscience.gov/blog/)
+- [code.gov blog](https://medium.com/@CodeDotGov)
+- [challenge.gov blog](https://www.challenge.gov/blog/)
```

Comment:
Is there an order to this list? Maybe alphabetical would be better? 🤷

Reply:
Copied the order from https://docs.google.com/document/d/1drX3O-qvkxdptS4d8W4TEuRpx3sUHaasENzvT-clDOc/edit. I'll ping Dahianna to see if the order matters.

Reply:
Alphabetical is fine. Let's also add the GSA Blog as an option: https://www.gsa.gov/blog